src/Module/WebFinger.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\Hook;
+use Friendica\Core\L10n;
+use Friendica\Core\Renderer;
+use Friendica\Core\System;
+use Friendica\Model\User;
+use Friendica\Network\Probe;
+use Friendica\Protocol\Salmon;
+use Friendica\Util\Strings;
+
+/**
+ * Prints information about people
+ * @see https://tools.ietf.org/html/rfc7033
+ */
+class WebFinger extends BaseModule
+{
+	public static function content()
```

Comment:
When does this module reach this function? What is the route that doesn't end up in `rawContent` and exits?

Reply:
I'd say in case there is no `$_GET['uri']` nor is it `/xrd`. But I think I've to split this into
`/.well-known/webfinger` and `/xrd` I guess